doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_matter_thread.rst

```diff
@@ -281,7 +281,7 @@ Matter and Thread support has the following limitations on the nRF54H20 DK:
 * The ``west flash --erase`` command is blocked.
   See :ref:`ug_nrf54h20_gs_sample` for more information.
 * The factory reset functionality does not work properly.
-  After clearing all NVM storage, the device can not reboot automatically and falls into a hard fault.
+  After clearing all NVM storage, the device cannot reboot automatically and falls into a hard fault.
```

Comment:
This file is not valid anymore here. It was used for the previous customer sampling releases and it is obsolete, Limitations there are not valid anymore. 
@canisLupus1313 @maciejbaczmanski Do you agree to remove it at all or at least modify it?

Reply:
This page is in orphaned state (in the repo) and not part of the public documentation. cc @FrancescoSer @annwoj

Reply:
What does it mean that it is not part of the public documentation? Will it not be visible at all?

Reply:
I would say, if it is available for users it will be misleading because many entries here are obsolete and not valid for Matter or Thread anymore. Unfortunately, I don't know how we should manage the orphaned pages like this one.

Reply:
Since it's orphaned, it won't be visible in the doctree. For the Moonlight we ended up removing the entire page (same article) and I believe you have adjusted the applicable content within the Matter documentation. Maybe the same can be done here? Even if it's not done for 2.7, it shouldn't be a problem as the page is orphaned.

Reply:
So what do you think @maciejbaczmanski @canisLupus1313? Are we going to remove it at all, or remove obsolete parts, leave architecture one and go public with it?

Reply:
I just checked, and on the TechDocs platform, orphaned pages are not available through the search, but can be accessed if you already know the full address to the page.

No other page links to this page, which would be another way to access it.

Reply:
I think this file was copied from ncs-next. And can be removed. Also the highlighted statement is not true since factory reset and reset work.

Reply:
That's true. So @umapraseeda could you please remove the whole page to be consistent with our current state?

Reply:
> So, if customers saved the page address during the first sampling they can reopen it. I think we can remove all sections starting from `Matter over Wi-Fi` to the end of the page and it will be consistent with what we currently have.

Even that wouldn't work, since the customer sampling documentation wasn't available on the new docs platform.

Reply:
So do you recommend doing nothing with that and leaving it as it is? I'm worried that we forget about it and it will go public without change.

Reply:
If the content is completely obsolete, I see no reason for keeping the page in the repo, we should remove it. Tagging @FrancescoSer  and @LuDuda  for visibility.

Reply:
I'll remove the file

Reply:
OK, thank you all for the explanation and help with that.